### ISSUE TITLE
Fetch resources by SVN

### ISSUE THREAD

**Issue by zhujian7** [tektoncd/pipeline#2361]
Tekton supports Git type PipelineResource perfectly, any plan to support SVN Resource?

**Comment by sbwsg** [comment]
Thanks for creating an issue!  I don't think we'd consider an SVN PipelineResource because PipelineResources are remaining in an alpha state and may at some point be deprecated.  We would gladly accept SVN Tasks in the [Tekton Catalog](https://github.com/tektoncd/catalog/) though!

We have a `git-clone` Task already that provides the same functionality as a Git PipelineResource.  It's available here: https://github.com/tektoncd/catalog/tree/v1beta1/git

**Comment by zhujian7** [comment]
@sbwsg  Glad to know, thanks. And I'm a little confused:

> because PipelineResources are remaining in an alpha state and may at some point be deprecated.

Will PipelineResource be **removed** and replaced by a task or **redesigned**  in the future?
If redesigned, what will it be responsible for?

**Comment by sbwsg** [comment]
We don't have any specific plans to remove `PipelineResources`.  Instead we've announced that they're not in Beta.  This means they don't get the same level of support as the stuff which is in Beta.  There's still a lot of discussions revolving around `PipelineResources` so no one really knows what's going to happen to them.  One of the reasons for keeping them alpha is because we haven't decided where they're headed.

> If redesigned, what will it be responsible for?

If we redesigned them then part of that process would require us to figure out and document what they should be responsible for.

I think we can do a better job documenting all this so I'll try to get some docs together soon to describe the current problems and possible next steps.
